src/main/java/org/takes/rq/RqForm.java

```diff
@@ -356,4 +362,3 @@ private static String encode(final CharSequence txt) {
         }
     }
 }
-
```

Comment:
There were two empty lines before. One empty line now. Github not shows last empty line when you view the file (you can check other files, to prove it). But if you view it on your local file system - file is OK: one empty line at the end of file.